File: app/src/main/java/com/example/jiun/sookpam/server/SearchableService.java
package com.example.jiun.sookpam.server;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface SearchableService {
    @GET("{query}")
    Call<List<RecordResponse>> getItems(@Path("query") String query);
}

File: app/src/main/java/com/example/jiun/sookpam/server/SearchableService.kt
package com.example.jiun.sookpam.server

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface SearchableService {
    @GET("search/{query}")
    fun getItems(
            @Path("query") query: String
    ): Call<List<RecordResponse>>
}
